microprofile.jdt/org.eclipse.lsp4mp.jdt.core/plugin.xml

```diff
@@ -47,6 +50,12 @@
       <provider class="org.eclipse.lsp4mp.jdt.internal.core.providers.MicroProfileProjectLabelProvider" />
    </extension>
 
+   <!-- Configuration sources -->
+   <extension point="org.eclipse.lsp4mp.jdt.core.configSourceProviders">
+      <provider class="org.eclipse.lsp4mp.jdt.internal.core.providers.DefaultMicroProfilePropertiesConfigSourceProvider" />
+      <provider class="org.eclipse.lsp4mp.jdt.internal.core.providers.QuarkusConfigSourceProvider" />
```

Comment:
QuarkusConfigSourceProvider should be hosted in quarkus-ls project but perhaps you want to do that in an another PR?

Reply:
Yes, I was planning to do this in another PR, since moving the implementation means rewriting a lot of the test cases. Also, I need the SNAPSHOT of this build to be published in order to recognize the extension point in quarkus-ls